Makefile

```diff
@@ -150,3 +148,14 @@ phpcs:
 ## Fix codebase according to Drupal standards https://www.drupal.org/docs/develop/standards
 phpcbf:
 	@$(call phpcsexec, phpcbf)
+
+## Add symbolic link from custom script(s) to /.git/hooks/pre-commit
+hooksymlink:
```

Comment:
This needs to point a file to check

Reply:
Updated

Reply:
Follow up should consider checking for target using make https://stackoverflow.com/questions/35588153/where-does-make-store-its-cache/35590775#35590775